apps/storybook/src/Separator.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Flex, Separator, Text } from "@optiaxiom/react";
+
+const meta: Meta<typeof Separator> = {
+  component: Separator,
+  title: "Components / Separator",
```

Comment:
let's move this inside the `data-display` directory and remove this explicit `title` property

Reply:
Done in https://github.com/optimizely-axiom/optiaxiom/pull/65/commits/f0e78d7fa1c3c4e4d059679a6f6e31fdb7577946.